packages/video_player/video_player/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 2.2.9
+
+* Adds compatibility with `video_player_platform_interface` 5.0, which does not
```

Comment:
Should we also mentioned the CONTRIBUTING.md change here?

Reply:
It seemed extremely niche (the audience would be "people who have contributed to `video_player` in the past, but don't read our current contributing docs"), so I didn't think it was worth mentioning.